source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -39,13 +39,18 @@ Some extra tips:
  - Home Assistant is installed in a virtual Python environment at `src/homeassistant`
  - Home Assistant will be started as a service run by the user `homeassistant`
 
+Accessing Your Raspberry Pi via SSH:
+
+ - Connect using Terminal.app or iTerm.app on a Mac, Putty on Windows, or other Terminal application via entering `ssh pi@XXX.XXX.XXX.XXX` (where X = your Raspberry Pi's local IP address).
```

Comment:
IP address in most of the documentation is written as IP-ADDRESS. Should probably be changed to stay consistent.